.github/workflows/ruby.yml

```diff
@@ -0,0 +1,27 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rails_version: [5.2.4.3, 6.0.3.2]
+        ruby: [2.7]
```

Comment:
This is a bit overkill since we're only using one version of ruby, but might be helpful to have this when we want to migrate to ruby 3 🤷